guide/additional-info/changes-in-v12.md

```diff
@@ -548,6 +549,16 @@ There have been several changes made to the `ClientOptions` object located in `c
 
 `clientOptions.sync` has been removed entirely, along with all other user account-only properties and methods.
 
+#### ClientOptions#disabledEvents
+
+`clientOptions.disabledEvents` has been removed in favor of using intents
+
+```diff
+- const client = new Discord.Client({ disabledEvents: [] })
++ const intents = new Discord.Intents([]);
```

Comment:
This would compute empty intents, causing the bot to not emit any events, the guide should not suggest that as replacement here, instead this should have a note leading to the section about to be added with #354 explaining the process of computing intents and what it entails in much more detail than possible in this short update part.

This section should also (shortly) mention that specifying intents may break the internal capabilities of the library and that users should only opt out of events if they know what they're getting into